src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -183,6 +185,12 @@ public void setCardinality(final O obj, final int count) {
         UnmodifiableCollection.unmodifiableCollection(new ArrayList<Object>());
 
     /**
+     * Random object used by random method. This has to be not local to the
+     * random method so as to not return the same value in the same millisecond.
+     */
+    private static final Random RANDOM = new Random();
```

Comment:
It will make the class more deterministic (and testable) if RANDOM is supplied with a seed at construction.